frontend/src/component/common/FormTemplate/FormTemplate.tsx

```diff
@@ -72,12 +76,14 @@ const StyledMobileGuidanceWrapper = styled('div', {
         : {}),
 }));
 
-const StyledMain = styled('div')(({ theme }) => ({
+const StyledMain = styled('div', {
+    shouldForwardProp: (prop) => prop !== 'useFixedSidebar',
+})<{ useFixedSidebar?: boolean }>(({ theme, useFixedSidebar }) => ({
     display: 'flex',
     flexDirection: 'column',
     flexGrow: 1,
     flexShrink: 1,
-    width: '100%',
+    width: useFixedSidebar ? 'unset' : formTemplateFormWidth,
```

Comment:
since width is not inherited can it be set to `initial` instead of 'unset'? According to this definition to understand unset you need to perform if/else in your head: "The unset CSS keyword resets a property to its inherited value if the property naturally inherits from its parent, and to its [initial value](https://developer.mozilla.org/en-US/docs/Web/CSS/initial_value) if not"

Reply:
Yeah, great point.